bucket/ungoogled-chromium.json

```diff
@@ -1,35 +1,51 @@
 {
-    "version": "79.0.3945.117-1",
-    "description": "Modifications to Google Chromium for removing Google integration and enhancing privacy, control, and transparency.",
+    "version": "79.0.3945.117",
+    "description": "Modifications to Google Chromium for removing Google integration and enhancing privacy, control, and transparency (macchrome's build)",
     "homepage": "https://github.com/Eloston/ungoogled-chromium",
```

Comment:
Shouldn't we also change the homepage to "https://github.com/macchrome/winchrome"?